src/state_machine/coordinator/mod.rs

```diff
@@ -503,6 +504,42 @@ pub mod test {
         assert_eq!(coordinator.get_state(), State::DkgPublicGather);
     }
 
+    pub fn start_signing_round<Coordinator: CoordinatorTrait>(sign_id: Option<u64>) {
```

Comment:
nit: for this test, I'm wondering if we should set the coordinator `sign_id` to some known value, ideally bigger than the one used in the test, and checking the effect of `start_signing_round`. So that we can test that we don't care about the ordering of the IDs, and that if none is passed then we increment the previously used one.